### PR TITLE
Website improvements/additions

### PR DESCRIPTION
**Pull request by hendrikbl**
Here's the pull request with all my changes and additions, as promised.

What has changed:
- News section layout
- changed hero-area images
- added navigation between posts
- added listing of all posts
- replaced lightbox with swipebox
- updated jquery + depending scripts
- added RSS feed
- various small improvements

As always you can have a look at it [here](http://hendrikbl.github.io/Vocaluxe/).

Let me know what you think :)

### CODE REVIEW

**Comment by flokuep** [comment]
Really like your changes - Thanks for working on this. I guess we need to release a new vocaluxe version in some near future to show this nice website to some vocaluxe users...

There is a little merge conflict in readme.md that need to be fixed; Do we use new.vocaluxe.org as url now?

**Comment by lukeIam** [comment]
I would say we use new.vocaluxe.org from now. And when we will go live i'll
change it to vocaluxe.org

**Comment by flokuep** [comment]
Okay, solved merge conflict, so we're ready for merging this.
👍

**Comment by hendrikbl** [comment]
> Okay, solved merge conflict, so we're ready for merging this.

Nice, thank you :)

File: js/main.js [hunk 1]
```diff
@@ -1,10 +1,17 @@
 //Contact Form
+$('#submit').click(function() {
```
Comment (lukeIam) [inline js/main.js]: @hendrikbl 
Do we use this?

File: css/ie.css [hunk 2]
```diff
@@ -0,0 +1,5 @@
+@media screen and (-ms-high-contrast: active), screen and (-ms-high-contrast: none) {
+  #hero-area {
+      background-image: none;
```
Comment (lukeIam) [inline css/ie.css]: is there a special reason why we don't show the background image for IE users?
At least with IE11 the background looks good (when I remove this line)

File: _posts/2012-07-04-first-final-release.md [hunk 3]
```diff
@@ -4,14 +4,21 @@ section-type: post
...
 
 You don't want to wait for the next final release and want to follow the latest edge of development instead? Just follow the developers on the Github Vocaluxe repository!
-[![Song Selection]({{site.baseurl}}/img/posts/release-01_Screenshot_SongSelection_Small.png){:class="img-responsive"}]({{site.baseurl}}/img/posts/release-01_Screenshot_SongSelection.png){:class="img-responsive" data-lightbox="screenshots" data-title="Song Selection"}
+
+## Gallery
+
+{:class="post-gallery"}
+
+<!-- [![Song Selection]({{site.baseurl}}/img/posts/release-01_Screenshot_SongSelection.png){:class="img-responsive"}]({{site.baseurl}}/img/posts/release-01_Screenshot_SongSelection.png){:class="img-responsive" data-lightbox="screenshots" data-title="Song Selection"} -->
+[![Song Selection]({{site.baseurl}}/img/posts/release-01_Screenshot_SongSelection.png){:class="img-responsive"}]({{site.baseurl}}/img/posts/release-01_Screenshot_SongSelection.png){:class="swipebox" title="Song Selection"}
```
Comment (lukeIam) [inline _posts/2012-07-04-first-final-release.md]: with IE11 the gallery preview image is much to big.
a 
``` css
width: 100%;
```
for IE11 helps.

File: _includes/news.html [hunk 4]
```diff
@@ -2,59 +2,59 @@
...
-                {% endif %}
-                </div>
+<section id="{{ page.section-type }}" class="content-section">
+  <div class="container">
+    <div class="row">
+      <div class="col-md-12">
+        <h1 class="title">{{ page.title }}</h1>
+        {% if post_1 != nil %}
+        <div class="col-md-8">
+          <div class="col-lg-12 news-card">
```
Comment (lukeIam) [inline _includes/news.html]: suggestion: make the complete card click able (at least for mobile devices)
My first intension on a touch device was to click the images...

File: css/ie.css (hunk 2 above)
Comment (hendrikbl) [inline css/ie.css]: IE doesn't support `background-blend-mode` as a css property so you can barely read the text. This is what it looks like for me in IE11:

grafik

That's why I just don't show it there. But of course we could just pick another image or simply darken the image itself.

File: js/main.js (hunk 1 above)
Comment (hendrikbl) [inline js/main.js]: I don't think so since there is no contact form :p

File: _posts/2012-07-04-first-final-release.md (hunk 3 above)
Comment (hendrikbl) [inline _posts/2012-07-04-first-final-release.md]: I just added it for every browser. Doesn't change anything.

File: _includes/news.html (hunk 4 above)
Comment (hendrikbl) [inline _includes/news.html]: I'll make the image clickable, I think the whole card would be too much...

**Comment by hendrikbl** [comment]
I had a look at the changes you mentioned. 
Here's what i changed: 74f31e7

File: css/ie.css (hunk 2 above)
Comment (lukeIam) [inline css/ie.css]: > IE doesn't support background-blend-mode as a css property so you can barely read the text.
> That's why I just don't show it there. 

I see...

We could try this for IE:

```css
@media screen and (-ms-high-contrast: active), screen and (-ms-high-contrast: none) {
    #hero-area:before {
        content: "";
        position: absolute;
        top: 0;
        bottom: -30px;
        width: 100%;
        background: rgba(44, 62, 80, 0.7); 
    }
}
```

it's not exactly the same but close enough thing 😉

File: _includes/news.html (hunk 4 above)
Comment (lukeIam) [inline _includes/news.html]: ok 👍

File: css/ie.css (hunk 2 above)
Comment (lukeIam) [inline css/ie.css]: @hendrikbl I cloun't get the IE workaround running... For now there is now BG image for IE...

**Pull request merged**